src/openrct2-ui/windows/Options.cpp

```diff
@@ -194,165 +194,165 @@ static constexpr const int32_t WH = 332;
+    MakeWidget({288,  54}, { 11, 10}, WWT_BUTTON,   1, STR_DROPDOWN_GLYPH,      STR_AUDIO_DEVICE_TIP ),
+    MakeWidget({ 10,  69}, {220, 12}, WWT_CHECKBOX, 1, STR_MASTER_VOLUME,       STR_MASTER_VOLUME_TIP), // Enable / disable master sound
+    MakeWidget({ 10,  84}, {220, 12}, WWT_CHECKBOX, 1, STR_SOUND_EFFECTS,       STR_SOUND_EFFECTS_TIP), // Enable / disable sound effects
+    MakeWidget({ 10,  99}, {220, 12}, WWT_CHECKBOX, 1, STR_RIDE_MUSIC,          STR_RIDE_MUSIC_TIP   ), // Enable / disable ride music
+    MakeWidget({ 10, 113}, {290, 13}, WWT_CHECKBOX, 1, STR_AUDIO_FOCUS,         STR_AUDIO_FOCUS_TIP  ), // Enable / disable audio disabled on focus lost
+    MakeWidget({ 10, 128}, {145, 13}, WWT_LABEL,    1, STR_OPTIONS_MUSIC_LABEL                       ), // Title music label
+    MakeWidget({155, 127}, {145, 13}, WWT_DROPDOWN, 1                                                ), // Title music
+    MakeWidget({288, 128}, { 11, 11}, WWT_BUTTON,   1, STR_DROPDOWN_GLYPH,      STR_TITLE_MUSIC_TIP  ),
+    MakeWidget({155,  68}, {145, 13}, WWT_SCROLL,   1, SCROLL_HORIZONTAL                             ), // Master volume
+    MakeWidget({155,  83}, {145, 13}, WWT_SCROLL,   1, SCROLL_HORIZONTAL                             ), // Sound effect volume
+    MakeWidget({155,  98}, {145, 13}, WWT_SCROLL,   1, SCROLL_HORIZONTAL                             ), // Music volume
     { WIDGETS_END },
 };
 
 static rct_widget window_options_controls_and_interface_widgets[] = {
     MAIN_OPTIONS_WIDGETS,
 #define CONTROLS_GROUP_START 53
-    { WWT_GROUPBOX,         1,  5,      304,    CONTROLS_GROUP_START + 0,    CONTROLS_GROUP_START + 91,   STR_CONTROLS_GROUP,                     STR_NONE },                                 // Controls group
-    { WWT_CHECKBOX,         2,  10,     299,    CONTROLS_GROUP_START + 13,   CONTROLS_GROUP_START + 26,   STR_SCREEN_EDGE_SCROLLING,              STR_SCREEN_EDGE_SCROLLING_TIP },            // Edge scrolling
-    { WWT_CHECKBOX,         2,  10,     299,    CONTROLS_GROUP_START + 30,   CONTROLS_GROUP_START + 41,   STR_TRAP_MOUSE,                         STR_TRAP_MOUSE_TIP },                       // Trap mouse
-    { WWT_CHECKBOX,         2,  10,     299,    CONTROLS_GROUP_START + 45,   CONTROLS_GROUP_START + 56,   STR_INVERT_RIGHT_MOUSE_DRAG,            STR_INVERT_RIGHT_MOUSE_DRAG_TIP },          // Invert right mouse dragging
-    { WWT_CHECKBOX,         2,  10,     299,    CONTROLS_GROUP_START + 60,   CONTROLS_GROUP_START + 71,   STR_ZOOM_TO_CURSOR,                     STR_ZOOM_TO_CURSOR_TIP },                   // Zoom to cursor
-    { WWT_BUTTON,           1,  155,    299,    CONTROLS_GROUP_START + 75,   CONTROLS_GROUP_START + 87,   STR_HOTKEY,                             STR_HOTKEY_TIP },                           // Set hotkeys buttons
+    MakeWidget({  5,  CONTROLS_GROUP_START + 0}, {300, 92}, WWT_GROUPBOX, 1, STR_CONTROLS_GROUP,          STR_NONE                       ), // Controls group
```

Comment:
```suggestion
    MakeWidget({  5,  CONTROLS_GROUP_START + 0}, {300, 92}, WWT_GROUPBOX, 1, STR_CONTROLS_GROUP,                                         ), // Controls group
```

Reply:
```suggestion
    MakeWidget({  5,  CONTROLS_GROUP_START + 0}, {300, 92}, WWT_GROUPBOX, 1, STR_CONTROLS_GROUP                                          ), // Controls group
```

(Posting primarily to commit through GitHub.)